src/cascadia/CascadiaPackage/Resources/en-US/Resources.resw

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This file changed because we no longer run the script before every build... so it's gotta be finalized.